buildtools/libinjection/Dockerfile

```diff
@@ -3,12 +3,13 @@
 
 FROM ghcr.io/corazawaf/coraza-proxy-wasm/buildtools-wasi-sdk:main
 
-RUN apt-get install -y patch
+RUN apt-get install -y autogen autoconf automake libtool
 
-RUN mkdir -p /libinjection && curl -L https://github.com/libinjection/libinjection/archive/49904c42a6e68dc8f16c022c693e897e4010a06c.tar.gz | tar -xz --strip-components 1 -C /libinjection
+RUN mkdir -p /libinjection && curl -L https://github.com/libinjection/libinjection/archive/bd86522711f6fd2b5f257023e19627fd6c1d5ca0.tar.gz | tar -xz --strip-components 1 -C /libinjection
```

Comment:
Shall we specify why this sha specifically?

Reply:
There isn't any reason then it being the latest at the time of writing, but such a comment would drift. We use shas in all dockerfiles as much as possible for deterministicness (especially since docker cache can really mess things up without)